README.md

```diff
@@ -1 +1,50 @@
 # HangOps Code of Conduct
+
+To continue growing a safe, welcoming, and fair community, all HangOps members are required to comply with the following code of conduct. HangOps admins are responsible for ensuring that all community members follow this code within HangOps spaces.
```

Comment:
> HangOps admins are responsible for ensuring that all community members follow 

In the section below on 'respectful behavior' the community is charged and empowered to call out disrespectful behavior. To me, this means that it is the community members collective responsibility of 'ensuring that all community members follow' through respectful deescalation and communication.

For the admins, it sounds like they are empowered to enforce the code and provide defense for the community if escalation by the community  members is needed.

Hopefully I am making sense. I think this sentence could be rephrased to say the members need to attempt respectful resolution of disrespect/abuses, and if that fails, the admins can be engaged to follow through.